### DISCUSSION TITLE
YouTube Vanced e YT music pararam

### DISCUSSION THREAD

**Discussion by JottaMCL** [TeamVanced/VancedManager#728] [Q&A]
Ambos os aplicativos pararam de funcionar. Acusam falta de internet. Já testei com versões anteriores e estão com o mesmo problema. 
Minha internet 4G está ok. Internet de fibra também. Já desinstalei tudo e reinstalei, mas problema persiste. Não sei mais o que fazer.

**Comment by Slav0282** [comment]


**Reply by JottaMCL** [reply]
Já sim, mas quando tento adicionar novamente, diz que já tenho a conta registrada.
